frontend/src/components/associations/Layout.tsx

```diff
@@ -0,0 +1,26 @@
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import { AssociationSidebar } from "./Sidebar";
+import Container from "react-bootstrap/Container";
+import React from "react";
+import { Association } from "../../models/associations/association";
+
+export const AssociationLayout = ({
```

Comment:
Maybe add a comment here to explain how to use this?

Reply:
For instance, I would personally like to know what kind of `additionalSidebar` should be passed on (with a separator, in a container, in a row, etc.)